src/components/SMSNotification/SMSNotification.css

```diff
@@ -42,6 +43,28 @@
     font-size: 1em;
 }
 
+.success {
+  color: green;
+  background-color: rgba(173, 255, 47, .3);
+  border-radius: 10px;
+  width: 90%;
+  padding: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 10px;
```

Comment:
These could be refactored with CSS below into a single rule for `.success, .error` to reduce duplicated code.

Reply:
Oh cool! I'll definitely do that.